src/components/Api/RecipientApi.js

```diff
@@ -1,39 +1,35 @@
+import axios from "axios";
+
 const BASE_URL = "https://rolling-api.vercel.app";
 
 export const getRecipient = async (recipientId) => {
-  const RESPONSE = await fetch(`${BASE_URL}/4-12/recipients/${recipientId}/`);
-
-  if (!RESPONSE.ok) {
-    throw new Error("사용자 정보를 받아오는데 실패했어요.");
-  }
+  const RESPONSE = await axios
+    .get(`${BASE_URL}/4-12/recipients/${recipientId}/`)
+    .catch((error) => {
+      throw new Error("사용자 정보를 받아오는데 실패했어요.");
+    });
```

Comment:
직접 정의한 에러보다 받은에러 자체를 그대로 던지는게 좋을 것 같습니다.
안그러면 디테일하게 에러를 핸들링할때 힘들 수 있어요!

```suggestion
    .catch((error) => {
      throw error;
    });
```